circle-mlir/circle-mlir/lib/dialect/mlir/CircleOps.td

```diff
@@ -296,6 +296,44 @@ class CIR_ConvOp<string mnemonic, string opSummary,
 //===----------------------------------------------------------------------===//
 // CIR op definitions.
 //===----------------------------------------------------------------------===//
+def CIR_AddOp : CIR_Op<"add", [
+    CIR_RuntimePredOpTrait<"Operands do not have valid shapes",
+      CPred<"Circle::VerifyAddOpShapeConstraints(llvm::cast<AddOp>($_op))">>,
+    ResultsBroadcastableShape,
+    DeclareOpInterfaceMethods<CIR_ShapeInferenceOpInterface>,
+    Pure,
+    Commutative,
+    // TODO enable QuantizableResult,
+    ]> {
+  let summary = "Addition operator";
+
+  let description = [{
+    Element-wise addition operation.
+  }];
+
+  let arguments = (
+    // TODO add more dtypes
+    ins CIR_TensorOf<[F32, I32, I64]>:$lhs,
+    CIR_TensorOf<[F32, I32, I64]>:$rhs,
```

Comment:
The reason why there's no Q8 or Q16 dtype here is because current circle-mlir focus on fake-quantized or float models?

Reply:
Yes...
But the actual reason is that quantize related codes are linked to TensorFlow that are too huge to import as of now.